deploy/olm-catalog/awx-operator/manifests/awx-operator.clusterserviceversion.yaml

```diff
@@ -167,6 +167,12 @@ spec:
         x-descriptors:
         - urn:alm:descriptor:com.tectonic.ui:advanced
         - urn:alm:descriptor:com.tectonic.ui:resourceRequirements
+      - displayName: PostgreSQL container storage requirements (when using a managed
+          instance)
```

Comment:
@fust can you make this a single line?

Reply:
Done. For consistency I also changed line 164